src/scripts/main.js

```diff
@@ -1,7 +1,37 @@
 'use strict';
 
 const pushNotification = (posTop, posRight, title, description, type) => {
-  // write code here
+  const notification = document.createElement('div');
+  const titleOfNot = document.createElement('h2');
+  const descriptionOfNot = document.createElement('p');
+
+  titleOfNot.textContent = title;
+  descriptionOfNot.textContent = description;
+
+  notification.classList.add('notification');
+  titleOfNot.classList.add('title');
+
+  notification.style.top = `${posTop}px`;
+  notification.style.right = `${posRight}px`;
+
+  notification.append(titleOfNot, descriptionOfNot);
+
+  switch (type) {
+    case 'success':
+      notification.classList.add(type);
+      break;
+    case 'error':
+      notification.classList.add(type);
+      break;
+    default:
+      notification.classList.add(type);
```

Comment:
Seems that you repeat the same action for each case. You can remove this sitch/case construction and just use ` notification.classList.add(type);`

Reply:
I don't see what causes overlapping providing in your screenshot.. it looks correct to me 

<img width="1400" alt="Screenshot 2023-06-20 at 1 46 26 PM" src="https://github.com/mate-academy/js_notification_DOM/assets/105941605/e0c9888a-4db3-494c-81aa-81da3c6ca972">